native/cocos/engine/Engine.h

```diff
@@ -139,6 +140,7 @@ class CC_DLL Engine : public BaseEngine {
     // Subsystems
     FileUtils *_fs{nullptr};
 	Profiler *_profiler{nullptr};
+    DebugRenderer *_debugRenderer{nullptr};
```

Comment:
Add #if CC_USE_PROFILER/#endif for _profiler declaration.

Reply:
It is fixed.